hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -1006,6 +1006,9 @@ int QuectelNcpClient::checkNetConfForImsi() {
         } else if (imsiCount >= IMSI_MAX_RETRY_CNT) {
             // if max retries are exhausted
             return SYSTEM_ERROR_AT_RESPONSE_UNEXPECTED;
+        } else if (r != AtResponse::OK && ncpId() == PLATFORM_NCP_QUECTEL_BG95_S5) {
+            // Ignore CIMI errors on BG95_S5, this means eSIM profile is not loaded
+            return SYSTEM_ERROR_NONE;
```

Comment:
LGTM!  

Can we add a "TODO: Remove or rework this if and when profile listing/switching API support is added to DVOS."